ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/TransactionPoolConfiguration.java

```diff
@@ -79,7 +81,15 @@ public static Builder builder() {
   public static class Builder {
     private int txPoolMaxSize = PendingTransactions.MAX_PENDING_TRANSACTIONS;
     private int pendingTxRetentionPeriod = PendingTransactions.DEFAULT_TX_RETENTION_HOURS;
-    private int txMessageKeepAliveSeconds = TransactionPool.DEFAULT_TX_MSG_KEEP_ALIVE;
+
+    @CommandLine.Option(
+        names = {"--Xincoming-tx-messages-keep-alive-seconds"},
+        paramLabel = "<INTEGER>",
+        hidden = true,
+        description =
+            "Keep alive of incoming transaction messages in seconds (default: ${DEFAULT-VALUE})",
+        arity = "1")
+    private Integer txMessageKeepAliveSeconds = TransactionPool.DEFAULT_TX_MSG_KEEP_ALIVE;
```

Comment:
Can this be a primitive `int` instead of an `Integer`?

Reply:
It does not work with an int. 
See: https://github.com/PegaSysEng/pantheon/blob/285f9f8589725355bc728b45402a7d5ee4562bde/pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java#L675
With an int the else is used and it tries to get a string and it fails.